Formula/dps8m.rb

```diff
@@ -31,29 +32,30 @@ def install
     # See https://sourceforge.net/p/dps8m/mailman/message/35960505/
     bin.mkpath
 
-    system "make", "INSTALL_ROOT=#{prefix}", "install"
+    system "make"
+    bin.install %w[dps8/dps8 punutil/punutil prt2pdf/prt2pdf].map { |f| "src/#{f}" }
```

Comment:
```suggestion
    bin.install %w[src/dps8/dps8 src/punutil/punutil src/prt2pdf/prt2pdf]
```

Simply prepending `src/` to these path strings is easier to read/understand/etc.